package.json

```diff
@@ -15,7 +15,7 @@
     "itest": "jest --config=itest/config.json --runInBand --verbose",
     "electron": "electron .",
     "build": "npm-run-all -p build:*",
-    "watch": "npm-run-all -p watch:*",
+    "watch": "npm-run-all -s clean build:* watch:*",
```

Comment:
looks like the `watch:*` scripts also run corresponding builds and there is some duplication here

Reply:
Yes, I can see the reason for curiosity. The watch scripts are the same as the build scripts but just pass an extra flag to them. They also "skip the initial" build, so that only watch for changes.

```
    "watch:js": "npm run build:js -- --watch --skip-initial-build",
    "watch:css": "npm run build:css -- --watch --skip-initial",
    "watch:static": "npm run build:static -- --watch",
```

I basically, added that initial build back in in the diff here. This way we can debug the main process quickly by opening two terminals and running:

**Terminal 1**
`npm run watch`

**Terminal 2**
`npm run electron` we can start and stop this one as much as we want and not have to wait for that initial build.